pkg/action/executor/k8s_controller.go

```diff
@@ -172,6 +192,40 @@ func (c *parentController) update(updatedSpec *k8sControllerSpec) error {
 	return err
 }
 
+func (c *parentController) rolloutDeploymentConfig(dc *osv1.DeploymentConfig) error {
+	// This will fail if the DC is already paused because of whatever reason
+	name := c.obj.GetName()
+	ns := c.obj.GetNamespace()
+	glog.V(3).Infof("Starting (Instantiating) the deploymentconfig %s/%s rollout", ns, name)
+	deployRequest := osv1.DeploymentRequest{Name: name, Latest: true, Force: true}
+	_, err := c.clients.osClient.AppsV1().DeploymentConfigs(ns).
+		Instantiate(context.TODO(), name, &deployRequest, metav1.CreateOptions{})
+	if err == nil {
+		glog.V(3).Infof("Rolled out (Instantiated) the deploymentconfig %s/%s", ns, name)
+	}
+
+	return err
```

Comment:
If the DC is paused, will the deploy request fail immediately? If so, will the error captures the information that the DC is paused? If not, shall we discover the paused state, and if it is true, log a warning and return immediately?